CIScripts/Common/Invoke-NativeCommand.Tests.ps1

```diff
@@ -0,0 +1,121 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestenvConfFile
+)
+
+. $PSScriptRoot/Init.ps1
+. $PSScriptRoot/../Testenv/Testenv.ps1
+. $PSScriptRoot/VMUtils.ps1
+
+. $PSScriptRoot/Invoke-NativeCommand.ps1
+
+Describe "Invoke-NativeCommand" {
+    Context "Local machine" {
+        It "does not throw on successful command" {
```

Comment:
Are we missing a test case where the command sends something to stderr but we don't end the world? (I think this is the primary reason for the existence of `Invoke-NativeCommand`)

Reply:
Do you know any builtin `exe` with such a behavior?

Reply:
Maybe we can simulate it like `whoami.exe 1>&2 `?